apps/client/src/app/career-qna/page.tsx

```diff
@@ -0,0 +1,87 @@
+'use client';
+
+import styled from '@emotion/styled';
+import { Flex, Stack, Text } from '@sickgyun/ui';
+import { useState } from 'react';
+import Footer from '@/components/common/Footer';
+import Header from '@/components/common/Header';
+import QnaAskBox from '@/components/qna-posting/QnaAskBox';
+import QnaListBox from '@/components/qna-posting/QnaListBox';
+import QnaPostingList from '@/components/qna-posting/QnaPostingList';
+import { QNA_LIST } from '@/constants/qna-list';
```

Comment:
```suggestion
import { QNA_LIST } from '@/constants/qna';
```

constants 파일명 qna-list 말고 qna라는 도메인 이름으로 묶어서 넣는것이 확장성을 고려하였을때 더 좋은 방법 같아용

Reply:
넵